slrealizer/Preprocessing+the+SDSS+non-lens+catalog.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
Shouldn't this notebook live in a "notebooks" folder?

Reply:
Moved to `demo` folder!